tests/integration/modules/test_cmdmod.py

```diff
@@ -1,634 +0,0 @@
-import os
-import random
-import sys
-import tempfile
-from contextlib import contextmanager
-
-import pytest
-
-import salt.utils.path
-import salt.utils.platform
-import salt.utils.user
-from tests.support.case import ModuleCase
-from tests.support.helpers import SKIP_INITIAL_PHOTONOS_FAILURES, dedent
-from tests.support.runtests import RUNTIME_VARS
-
-AVAILABLE_PYTHON_EXECUTABLE = salt.utils.path.which_bin(
-    ["python", "python2", "python2.6", "python2.7"]
```

Comment:
Not sure if I understood why the whole test file was dropped, but I could assume due to missing python executable. Or am I wrong?

Reply:
These tests were actually just migrated and moved to this other file: https://github.com/openSUSE/salt/pull/616/files#diff-7f58f80c7428a46156300ebfac3a3ee8d98811d3e8e69fd26a51239c2dc6ee29

(Backported from: https://github.com/saltstack/salt/pull/65679)